File: MyNewAsyncDemo/app/src/main/java/async/crash/com/mynewasyncdemo/MainActivity.java
package async.crash.com.mynewasyncdemo;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void callInBuilt(View view){

    }

    public void callCustom(View view){
        Intent i1 = new Intent();
        i1.setAction("com.")
    }
}

File: KotlinSeekbar/app/src/main/java/mjthornton/com/kotlinseekbar/MainActivity.kt
package mjthornton.com.kotlinseekbar

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.SeekBar
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val initialTextViewTranslationY = textview_progress.translationY

        seekBar.setOnSeekBarChangeListener(object: SeekBar.OnSeekBarChangeListener{
            override fun onProgressChanged(seekBar: SeekBar?, progress: Int, fromUser: Boolean) {
                textview_progress.text = progress.toString()

                val translationDistance = (initialTextViewTranslationY +
                        progress * resources.getDimension(R.dimen.text_anim_step) -1)
                textview_progress.animate().translationY(translationDistance)
            }

            override fun onStartTrackingTouch(seekBar: SeekBar?) {
            }

            override fun onStopTrackingTouch(seekBar: SeekBar?) {
            }
        })

        button_reset.setOnClickListener { v ->
            seekBar.progress = 0
            textview_progress.animate().setDuration(500).rotationBy(360f)
                    .translationY(initialTextViewTranslationY)
        }

    }
}
